docs/HOWTO-es.md

```diff
@@ -1,17 +1,34 @@
+# Primeros pasos
+
+<div align="right" markdown="1">
+
 *[Lea esto en otros idiomas](../README.md#translations)*
 
-¡Sea bienvenido a *Free-Programming-Books*! Damos una calurosa bienvenida a los nuevos colaboradores; incluso a aquellos que realizan su primera Pull Request (PR) en GitHub. Si es usted uno de ellos, aquí van algunos recursos que quizás le pueden ayudar:
+</div>
+
+**¡Sea bienvenido a `Free-Programming-Books`!**
```

Comment:
on monospacing repo title - not sure it adds much, maybe not worth the overhead needed to keep it uniform.

Reply:
Taking precedence I understand normally that monospace format means things non translatable strings.

The weird thing is that in one files are monospaced (with backticks), in others are rendered in cursive (`*word*` or `_word_`), in anothers are enclosed between quotes `""` and the last and most... without decoration.

Then the suggestion is... ¿? My preference... backticks or cursive

Reply:
In the context of translations, I see the benefit. backticks then

Reply:
Done between 0ee043dc9a6796a7a3fc2586968f2066a779ce1e and 7b599acb2490a6d5133133f7268aa354a0a46ce8